.changeset/pretty-badgers-sneeze.md

```diff
@@ -0,0 +1,5 @@
+---
+'@zoralabs/nft-drop-contracts': patch
+---
+
+Added pgn mainnet
```

Comment:
should this be one file?

Reply:
yeah it could be, but they could also be considered separate changesets.  either way i think is fine